draft-ietf-quic-transport.md

```diff
@@ -1232,6 +1232,19 @@ handshake data start from zero in each packet number space.
 Endpoints MUST explicitly negotiate an application protocol.  This avoids
 situations where there is a disagreement about the protocol that is in use.
 
+Implementations need to maintain a buffer of CRYPTO data received out of order.
+Because there is no flow control of CRYPTO frames, and endpoint could
+potentially force its peer to buffer an unbounded amount of data.
+Implementations MUST support buffering at least 4096 bytes of data.
```

Comment:
Does this requirement apply during the handshake?  Perhaps document some recommendations.

Being unable to buffer CRYPTO frames during the handshake leads to connection failure. Endpoints MAY choose to allow more data to be buffered during the handshake and reduce buffering once the handshake completes.  A larger limit during the handshake could allow for larger keys or credentials to be exchanged.

Reply:
Yes, it applies during the handshake.

I've updated the text to discuss buffer sizing and separate out pre- vs post-handshake completion behavior.